### PR TITLE
Customizable not-modified-char

### PR DESCRIPTION
**Pull request by jobbflykt**
Make the not-modified character customizable just like sml/modified-char.

### CODE REVIEW

File: smart-mode-line.el [hunk 1]
```diff
@@ -515,6 +515,13 @@ If this variable is nil, nothing is displayed."
   :group 'smart-mode-line-others
   :package-version '(smart-mode-line . "1.16"))
 
+(defcustom sml/not-modified-char (char-to-string
+                                  (if (char-displayable-p ?✓) ?✓ ? ))
```
Comment (Malabarba) [inline smart-mode-line.el]: Unfortunately, this character is not fixed-width. For instance, in my font (monaco) it's a little wider than a regular character, so the modeline moves around when the user saves/edits the buffer.

Just leave the default as `" "`

**Comment by Malabarba** [comment]
Thank for the PR. I'm in favor of the feature, I just think the default value needs to be kept as `" "`. 

Other than that, it's good to merge.

**Comment by jobbflykt** [comment]
I updated the change so that the default character is " ". I'm new to github but I guess the pull request should now be updated. Thanks for the feedback.

**Comment by Malabarba** [comment]
Looks good. Thanks!

**Pull request merged**